### PR TITLE
Readme.md - Small Fixes To Docs Section

### PR DESCRIPTION
**Pull request by war59312**
Small fix for Documentation section

### CODE REVIEW

**Comment by drwetter** [comment]
can a native speaker pls review this?

**Comment by teward** [comment]
I'm native American english speaker, i have a revision to request in this change and will submit when I am at a computer.  (this reply sent via phone)

File: Readme.md [hunk 1]
```diff
@@ -83,8 +83,7 @@ Support for 2.9.5 has been dropped. Supported is >= 3.0.x only.
 
 * .. it is there for reading. Please do so :-) -- at least before asking questions. See man page in groff, html and markdown format in `~/doc/`.
 * [https://testssl.sh/](https://testssl.sh/) will help to get you started.
-* For the (older) version 2.8 will Hunt provides a longer, good [description](https://www.4armed.com/blog/doing-your-own-ssl-tls-testing/), including useful background info.
-
+* For the (older) version 2.8, Will Hunt provides a longer, good [description](https://www.4armed.com/blog/doing-your-own-ssl-tls-testing/), including useful background information.
```
Comment (teward) [inline Readme.md]: Remove ", good" - its improper english and isnt necessary in the context its being used.

**Review by teward — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (war59312) [reply Readme.md]: Done

**Comment by drwetter** [comment]
Thanks!

**Pull request merged**